lib/tower_discord/reporter.ex

```diff
@@ -64,7 +64,7 @@ defmodule TowerDiscord.Reporter do
     do: {format_module(module), format_function(function, parameters)}
 
   defp format_module(module), do: Atom.to_string(module)
-  defp format_function(function, 0), do: Atom.to_string(function) <> "/0"
+  defp format_function(function, n) when is_integer(n), do: Atom.to_string(function) <> "/#{n}"
 
   defp format_function(function, parameters) when is_list(parameters),
```

Comment:
I think some of these function can be refactored to instead use utility functions in the native `Exception` module, but for now didn't want to touch and mix it with this PR.

Reply:
This is how I am refactoring `tower_slack` message format for what is worth: https://github.com/mimiquate/tower_slack/pull/36/files#diff-44e2ab7d8d63576e049b1ee94bae6fd420acce1eb20d4cd3f8ec65d263fc54be.